flask_appbuilder/tests/test_theme.py

```diff
@@ -0,0 +1,78 @@
+from flask_appbuilder import SQLA
+
+from .base import FABTestCase
+from .const import (
+    PASSWORD_ADMIN,
+    USERNAME_ADMIN,
+)
+
+
+class TestNoTheme(FABTestCase):
+    def setUp(self):
+        from flask import Flask
+        from flask_appbuilder import AppBuilder
+
+        self.app = Flask(__name__)
+        self.app.config.from_object("flask_appbuilder.tests.config_api")
+        self.db = SQLA(self.app)
+        self.appbuilder = AppBuilder(self.app, self.db.session)
+
+    def test_properties(self):
+        self.assertEqual(self.appbuilder.app_theme, "")
+        self.assertEqual(self.appbuilder.app_theme_custom, "")
+
+    def test_view(self):
+        self.assertGreater(len(self.appbuilder.baseviews), 0)
+        client = self.app.test_client()
+        self.browser_login(client, USERNAME_ADMIN, PASSWORD_ADMIN)
+        rv = client.get("/users/list/")
+        self.assertEqual(rv.status_code, 200)
+
+
+class TestTheme(FABTestCase):
+    def setUp(self):
+        from flask import Flask
+        from flask_appbuilder import AppBuilder
+
+        self.app = Flask(__name__)
+        self.app.config.from_object("flask_appbuilder.tests.config_api")
+        self.app.config["APP_THEME"] = "lumen.css"
+        self.db = SQLA(self.app)
+        self.appbuilder = AppBuilder(self.app, self.db.session)
+
+    def test_properties(self):
+        self.assertEqual(self.appbuilder.app_theme, "lumen.css")
+        self.assertEqual(self.appbuilder.app_theme_custom, "")
+
+    def test_view(self):
+        self.assertGreater(len(self.appbuilder.baseviews), 0)
+        client = self.app.test_client()
+        self.browser_login(client, USERNAME_ADMIN, PASSWORD_ADMIN)
+        rv = client.get("/users/list/")
+        self.assertEqual(rv.status_code, 200)
+
+
+class TestCustomTheme(FABTestCase):
+    def setUp(self):
+        from flask import Flask
+        from flask_appbuilder import AppBuilder
+
+        self.app = Flask(__name__)
+        self.app.config.from_object("flask_appbuilder.tests.config_api")
+        self.app.config["APP_THEME_CUSTOM"] = "/static/appbuilder/css/themes/lumen.css"
+        self.db = SQLA(self.app)
+        self.appbuilder = AppBuilder(self.app, self.db.session)
+
+    def test_properties(self):
+        self.assertEqual(self.appbuilder.app_theme, "")
+        self.assertEqual(
+            self.appbuilder.app_theme_custom,
+            "/static/appbuilder/css/themes/lumen.css"
+        )
+
+    def test_view(self):
+        self.assertGreater(len(self.appbuilder.baseviews), 0)
+        client = self.app.test_client()
+        self.browser_login(client, USERNAME_ADMIN, PASSWORD_ADMIN)
+        rv = client.get("/users/list/")
+        self.assertEqual(rv.status_code, 200)
```

Comment:
It would be great to actually assert `/static/appbuilder/css/themes/lumen.css` is included on the view on all `test_view`'s

Reply:
made these changes